README.md

```diff
@@ -276,6 +276,13 @@ The default value is `'keep'`.
   - `'keep'` The session in the store will be kept, but modifications made during
     the request are ignored and not saved.
 
+##### signature
+
+Pass in your own cookie signing object/module here that implements the
+`sign(value, secret)` and `unsign(value, secret)` functions.
+
+The default value is the `node-cookie-signature` module.
```

Comment:
Typo here, as this module uses the `cookie-signature` module, not `node-cookie-signature`.